vmsyscall/src/lib.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! syscall serialize/deserialize
+//!
+//! Currently it uses a hard coded page and an I/O trigger.
+//! We might want to switch to MMIO.
+
+#![deny(missing_docs)]
+#![deny(clippy::all)]
+#![no_std]
+
+/// The IO trigger port for vm syscall proxy
+pub const TRIGGER_PORT: u16 = 0xFF;
```

Comment:
Sorry for one more question. `TRIGGER_PORT` seems to me a VM specific detail which we won't use in process-based TEEs. Is this crate the right place for it?

Reply:
oh .. right..

Reply:
removed